shared_workflow/install_shared.py

```diff
@@ -92,25 +92,6 @@ def install_simulation(
         FaultParams.vel_mod_dir.value: vel_mod_dir,
     }
 
-    # VM params
-    vm_params_dict = {
-        VMParams.gridfile.value: os.path.join(
```

Comment:
These aren't actually saved to the vm config in srfinfo2vm. Are they written elsewhere before install?

Reply:
its generated by gen_coords

Reply:
i'm moving them to gen_coords, or at least somewhere in Pre-processing. ( since its gen_coords that generates them)
the current workflow doesnt make sense and cause unexpected file corruptions.
what we have now in install_cybershake_fault:
1. validate_vm
2.install sim folder
2a. hard-code and  then change/dump yaml config file (invalids the validation just happened)

IMO, the install process should not be touching files in Data/Model folder
What raised this issue is that there was a crash during install phase, and it corrupted/nuked the yaml file with bad info

Reply:
Sweet. I'll resolve this conversation once the pull-request in pre-pro is merged.
Is gen-coords going to be called from srfinfo2vm? Or what are you thinking?

Reply:
gen_coords is already called by srfinfo2vm, but currently only generates the files (e.g gridfile, gridout etc) without updating the vm_params.yaml.
I'm thinking adding in the update function in gen_coords.

Reply:
https://github.com/ucgmsim/Pre-processing/pull/173